pkgs/development/python-modules/playwright/driver.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchurl
+, playwrightVersion
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  selectSystem = attrs: attrs.${system} or (throw "Unsupported system: ${system}");
+  suffix = selectSystem {
+    x86_64-linux = "linux";
+    aarch64-linux = "linux-arm64";
+    x86_64-darwin = "mac";
+    aarch64-darwin = "mac-arm64";
+  };
+  sha256 = selectSystem {
+    x86_64-linux = "sha256-RyTYrIBMIFuIfr/WRH+qXf3DcQvEDLBePD1PeKzY93c=";
+    aarch64-linux = "sha256-u5OtIVygcVQALqxWO/bEhf7jaGC6PtY/1sqrb5GH/kM=";
+    x86_64-darwin = "sha256-KTg4nt/MZL7PzBCsU0Y9Pq9j2QEbG3iOdhWdC66romQ=";
+    aarch64-darwin = "sha256-xdrWn+fZC0QbUe3uCqWErtTaSKWOIwp+0zLLSFXnA/A=";
+  };
```

Comment:
Is there a reason that the driver is in its own file? I find it a bit weird to have a `playwrightVersion` option here but use hardcoded SHA256s anyway. What would you think about moving this (or the entire driver) into `default.nix`?